Add warning while creating keytab entry (#1958)
theforeman/foreman-documentation: guides/common/modules/proc_configuring-direct-ad-integration-with-gss-proxy.adoc

----
. Create a keytab entry:
+

> WARNING: While creating a keytab entry, a host should not be connected to a domain.

Reviewer: ```suggestion
WARNING: Do not connect the host to a domain before creating a keytab entry.
```

Author: Better :+1:

Author: Waiting for technical review and even waiting to know if it is absolutely required.

Reviewer 2: I think the more correct term is "join" or "enroll", so maybe "The host should not be enrolled to a domain before creating…"?

That said my understanding of AD domains is very limited and I can not really say something about the change (it *sounds* plausible, but that's all).

> WARNING: The host should not be enrolled to a domain before creating a keytab entry.

Reviewer 3: Should not or must not?

Author: Yes :+1:
